src/main/flight/pid.c

```diff
@@ -664,15 +667,22 @@ void pidInitConfig(const pidProfile_t *pidProfile)
         thrustLinearizationB = (1.0f - thrustLinearization) / (2.0f * thrustLinearization);
     }
 #endif    
-#if defined(USE_D_CUT)
-    dtermCutPercent = pidProfile->dterm_cut_percent / 100.0f;
-    dtermCutPercentInv = 1.0f - dtermCutPercent;
-    dtermCutRangeHz = pidProfile->dterm_cut_range_hz;
-    dtermCutLowpassHz = pidProfile->dterm_cut_lowpass_hz;
-    dtermCutGain = pidProfile->dterm_cut_gain * dtermCutPercent * D_CUT_GAIN_FACTOR / dtermCutLowpassHz;
+#if defined(USE_D_MIN)
+    dMin[FD_ROLL] = (pidProfile->d_min_roll);
+    dMin[FD_PITCH] = (pidProfile->d_min_pitch);
+    dMin[FD_YAW] = 0;
```

Comment:
We should include the appropriate parameter for yaw. There's no point in having yaw axis support when the parameter is hardcoded to 0 disabling it. If we're going to support yaw D as a useful capability for thrust-based configurations (like tricopters for example), then we should be consistent and an D-related functionality.

Reply:
No problem, done.